assets/js/directives/contextual-message.directive.js

```diff
@@ -37,10 +37,13 @@ function contextualMessage($cookies, $window, Wallet, SecurityCenter, filterFilt
     };
 
     scope.shouldShow = () => {
-      let balance   = Wallet.total('');
-      let security  = SecurityCenter.security.score;
-      let isTime    = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
-      return balance > 20000000 && security < 0.5 && isTime;
+      let isTime = scope.msgCookie ? Date.now() > scope.msgCookie.when : true;
```

Comment:
msgCookie is the data from the contextual-message cookie. This is how we check to see how much time has gone by since the security warning has been dismissed.